wire-library/wire-tests/src/commonTest/kotlin/com/squareup/wire/KotlinMapTest.kt

```diff
@@ -40,6 +40,7 @@ class KotlinMapTest {
     assertNotNull(empty.things)
   }
 
+  @IgnoreJs
```

Comment:
Is there a bug we could file on youtrack?

Reply:
Maps casted to MutableMaps can be cleared. The test is exercising something JS isn’t particularly good at.

Reply:
Hang on, I have it wrong here. It’s because our unmodifiable functions don’t work on Native or JS.

```
@Suppress("NOTHING_TO_INLINE") // Syntactic sugar.
actual inline fun <K, V> MutableMap<K, V>.toUnmodifiableMap(): Map<K, V> = this
```

Reply:
(not going to fix this now, regardless)